### PR TITLE
fix: add media metadata via ffprobe-static

### PR DESCRIPTION
**Pull request by luwes**
related #22

ffprobe-static includes binaries for each platform
https://github.com/eugeneware/ffmpeg-static/tree/master/packages/ffprobe-static

### CODE REVIEW

File: src/next-video.tsx [hunk 1]
```diff
@@ -111,6 +114,7 @@ export default function NextVideo(props: NextVideoProps) {
         poster=""
         style={{
           '--controls': controls === false ? 'none' : undefined,
+          aspectRatio: aspectRatio !== '16/9' ? aspectRatio : undefined,
```
Comment (mmcc) [inline src/next-video.tsx:117]: What's the reasoning behind this check? We don't want to use the aspect ratio if it's `4:3` or otherwise?

Comment (luwes) [reply src/next-video.tsx:117]: the default is 16 / 9 defined in the inline style tag. if it's the same than 16/9 we skip setting it as an inline style

**Comment by heff** [comment]
Anything left to do on this one?

**Comment by luwes** [comment]
I think we decided not to pursue this for the time being. 
I'll keep it in the back pocket...

**Pull request closed without merging**